src/test/java/org/folio/permstest/TestUtil.java

```diff
@@ -55,43 +57,50 @@ public JsonObject getJson() {
   }
 
   public static Future<WrappedResponse> doRequest(Vertx vertx, String url,
-                                                  HttpMethod method, MultiMap headers, String payload,
-                                                  Integer expectedCode) {
+    HttpMethod method, MultiMap headers, String payload,
+    Integer expectedCode) {
     Promise<WrappedResponse> promise = Promise.promise();
```

Comment:
No need to create a Promise, you can directly return a Future.

Reply:
Fixed